### ISSUE TITLE
Help content creation

### ISSUE THREAD

**Issue by rodenhausen** [biosemantics/etc-site#201]
Help content for each Place has to be created so Help bar and help-"open in new window" can be activated.

**Comment by hongcui** [comment]
Yes help content needs to be created. This part needs to be integrated with Sanket's new assignment on user profile.
